js/assets/ohdsi.util.js

```diff
@@ -65,7 +65,7 @@ define(['jquery', 'knockout', 'lz-string', 'lodash', 'crossfilter'], function ($
 					if (obj.hasOwnProperty(key)) {
 						let path = `${currentPath}.${key}`.replace(/^\./, '');
 						if (typeof obj[key] !== 'undefined' && obj[key] !== null && typeof obj[key].subscribe === 'function') {
```

Comment:
I think this statement is detecting if the field is an observable, and for this, I'd use ko.isObservable() for this, described here: https://stackoverflow.com/questions/9625591/determine-if-an-object-property-is-ko-observable